scripts/vl3/vl3_interdomain.sh

```diff
@@ -48,7 +48,7 @@ for i in "$@"; do
         --wcmNsrPort=?*)
             WCM_NSRPORT=${i#*=}
             ;;
-        --delete)
+        --cleanup)
```

Comment:
In the interest of not breaking existing usage of this script, changes to params should be additive rather than removing/replacing existing params.  So, make this `--delete|--cleanup)` so either will work.

Reply:
Ok I'll do that.